### ISSUE TITLE
Enhance cyclic learning rate to include One Cycle policy

### ISSUE THREAD

**Issue by mdrmuhaimin** [pytorch/pytorch#19129]
## 🚀 Feature
In issue  #1909 which got resolved by #18001 implement Cycling learning. Lets enhance it or create a new Lr_scheduler to include one cycle policy

## Motivation
One cycle policy has proposed as a new enhancement to Cyclic Learning by same author https://arxiv.org/abs/1803.09820.  With one cycle policy we will have only 2 step size and it will have triangular cycle. This can now currently be done passing parameter to cyclic learning. However as this become popular day by day it is worthwhile to create it as a standalone function. Please share your opinion.

**Comment by alok** [comment]
I think this can be closed after https://github.com/pytorch/pytorch/commit/05f1fed693fe20d66730e593aa9efd70c7571fa5?
